packages/terra-core-docs/src/terra-dev-site/doc/icon/IconCollectionTable/clinical-lowlight-theme/IconCollectionTable.module.scss

```diff
@@ -0,0 +1,19 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-docs-icon-table-border-color: #9b9b9b;
+    --terra-docs-icon-table-header-cell-background-color: #0c0d0e;
+    --terra-docs-icon-table-subheader-cell-background-color: #2d3539;
+    --terra-docs-icon-table-cell-background-color: #1a2023;
+    // not for here
```

Comment:
can you explain what do you mean by this?

Reply:
Ah that was me pasting values here to reference and somehow I missed that it was still in here 😅